data/mods/innawood/npcs/missiondef.json

```diff
@@ -0,0 +1,421 @@
+      "success_lie": "This isn't what we need.",
+      "failure": "If only we could find a great valley or something."
+    }
+  },
+  {
+    "id": "MISSION_GET_RELIC",
+    "type": "mission_definition",
+    "name": { "str": "Find Rock" },
+    "goal": "MGOAL_FIND_ANY_ITEM",
+    "difficulty": 2,
+    "value": 100000,
+    "item": "rock",
+    "assign_mission_target": { "om_terrain": "forest", "reveal_radius": 3, "search_range": 120 },
+    "update_mapgen": { "place_item": [ { "item": "rock", "x": 14, "y": 15, "amount": [ 10, 20 ], "repeat": [ 1, 3 ], "chance": 20 } ] },
+    "origins": [ "ORIGIN_OPENER_NPC", "ORIGIN_ANY_NPC" ],
+    "dialogue": {
+      "describe": "St. Michael the archangel defend me in battle…",
+      "offer": "As the world seems to abandon the reality that we once knew, it becomes plausible that the old superstitions that were cast aside may have had some truth to them.  Please go and find me a religious relic… I doubt it will be of much use but I've got to hope in something.  It will look like a normal rock to your puny mind.",
+      "accepted": "I wish you the best of luck, may whatever god you please guide your path.",
+      "rejected": "Ya, I guess the stress may just be getting to me…",
+      "advice": "I hear rocks may be found in the woods.  They tell me things.  The spirits.  Religion!",
+      "inquire": "Any luck?  Please find me a small relic.  Any relic will do.  It looks like a rock.",
+      "success": "Thank you, I need some time alone now…",
+      "success_lie": "What good does this do us?",
+      "failure": "It was a lost cause anyways…"
+    }
+  },
+  {
+    "id": "MISSION_GET_SOFTWARE",
+    "type": "mission_definition",
+    "name": { "str": "Retrieve Stick" },
+    "goal": "MGOAL_FIND_ANY_ITEM",
+    "item": "stick",
+    "difficulty": 2,
+    "value": 80000,
+    "assign_mission_target": { "om_terrain": "forest", "reveal_radius": 3, "search_range": 120 },
+    "update_mapgen": { "place_item": [ { "item": "stick", "x": 14, "y": 15, "amount": [ 10, 20 ], "repeat": [ 1, 3 ], "chance": 20 } ] },
+    "origins": [ "ORIGIN_OPENER_NPC", "ORIGIN_ANY_NPC" ],
+    "dialogue": {
+      "describe": "Oh man, I can't believe I forgot to bring it…",
+      "offer": "There's some important notes on my special stick that I need.",
+      "accepted": "Thanks!  Just get the stick and bring it to me.",
+      "rejected": "Seriously?  It's an easy job…",
+      "advice": "Don't worry, you'll know the stick when you see it, it is unmistakeable.",
+      "inquire": "So, do you have my stick yet?",
+      "success": "Excellent, thank you!",
+      "success_lie": "What?!  You liar!",
+      "failure": "Wow, you failed?  All that work, down the drain…"
+    }
+  },
+  {
+    "id": "MISSION_GET_ZOMBIE_BLOOD_ANAL",
+    "type": "mission_definition",
+    "name": { "str": "Analyze Mushroom" },
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 8,
+    "value": 250000,
+    "item": "mushroom",
```

Comment:
This quest could be remade to be about finding alien fungus chunk instead, would perhaps make more sense.

Reply:
Agreed, that's much better. Done.